README.md

```diff
@@ -61,13 +61,18 @@ Please refer to [official doc](https://docs.docker.com/engine/install/).
 
 - Launch EVA database Server: `python eva.py`
 
-- Launch CLI: `python eva_client.py`
+- Launch CLI: `python eva_cmd_client.py`
 
-2. Run the `LOAD` command in the client terminal: (may take a while)
+2. Run the `UPLOAD` command in the client terminal:
 ```mysql
-LOAD DATA INFILE 'data/ua_detrac/ua_detrac.mp4' INTO MyVideo;
+UPLOAD INFILE 'data/ua_detrac/ua_detrac.mp4' PATH 'test_video.mp4';
 ```
-3. Below is a basic query that should work on the client
+
+3. Run the `LOAD` command in the client terminal: (may take a while)
+```mysql
+LOAD DATA INFILE 'test_video.mp4' INTO MyVideo;
+```
+4. Below is a basic query that should work on the client
 ```mysql
 SELECT id, data FROM MyVideo WHERE id < 5;
```

Comment:
Personally, I feel this query doesn't look good on the readme. It doesn't accomplish much. How about we replace it with the select query in tutorial? At least that has object detection. What do you think?

Reply:
But when a person is starting off, they need to know how to load a video right? for any select query to even work.

Reply:
Ah! I missed it is under the client testing. Then, it is fine.